apps/api/src/app/gene/gene.controller.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  Controller,
+  Get,
+  Param,
+  Query,
+} from '@nestjs/common';
+import { GeneService } from './gene.service';
+import {ModuleRef} from "@nestjs/core";
+
+@Controller('genes')
+export class GeneController {
+  constructor(private readonly geneService: GeneService) {}
+
+  @Get()
+  async getHugoGenes(@Query("query") query: string) {
+    try {
+      return await (await this.geneService.filterGeneSymbols(query)).slice(0,10);
```

Comment:
why are we using two `await` here?

Reply:
also try/catch should be done in the service, controllers should be clean and lean